projects/addon-editor/components/toolbar-tools/align-content/align-content.component.ts

```diff
@@ -32,11 +32,4 @@ export class TuiAlignContentComponent {
         @Inject(TUI_EDITOR_TOOLBAR_TEXTS)
         readonly texts$: Observable<LanguageEditor['toolbarTools']>,
     ) {}
-
-    private getActiveStatus$(status: string): Observable<boolean> {
-        return this.editor.stateChange$.pipe(
```

Comment:
let's add `startWith(null)` so we get the actual value immediately and drop `startWith` in other places

Reply:
maybe `startWith(false)` better ?

Reply:
No, because you don't know when you will subscribe. startWith(null) -> map makes the value actual at the time of subscription

Reply:
done